deploy.sh

```
@@ -0,0 +1,55 @@
+# DOCKER 로그인
+docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
+
+# 실행중인 컨테이너 확인
+if docker ps --format '{{.Names}}' | grep -q "$BLUE_CONTAINER"; then
+  CURRENT="blue"
+  IDLE="green"
+  IDLE_PORT=8081
+else
+  CURRENT="green"
+  IDLE="blue"
+  IDLE_PORT=8080
```

Shellcheck (0.10.0)

[warning] IDLE_PORT appears unused. Verify use (or export if used externally).

(SC2034)